### PR TITLE
Add rule for SLES-15-030380

### PR DESCRIPTION
**Pull request by teacup-on-rockingchair**
#### Description:

- Add rule for Ensure auditd Collects Information on the Use of Privileged Commands - insmod

#### Rationale:

- Add ansible remediation and tests
- Add rule concerning auditctl of insmod commnad to SLE15 stig profile
- Fix ansible lint error 'risky-file-permissions File permissions unset or incorrect' for ansible_audit_augenrules_add_watch_rule and ansible_audit_auditctl_add_watch_rule macros

### CODE REVIEW

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands_insmod/rule.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    unusual activity.
+
+severity: medium
+
+identifiers:
+    cce@sle15: CCE-85744-1
+
+references:
+    disa: CCI-000130,CCI-000169,CCI-000172,CCI-002884
+    nist@sle15: AU-12(c),AU-12.1(iv),AU-3,AU-3.1,AU-12(a),AU-12.1(ii),MA-4(1)(a)
```
Comment (vojtapolasek) [inline linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands_insmod/rule.yml:34]:
```suggestion
    nist: AU-12(c),AU-12.1(iv),AU-3,AU-3.1,AU-12(a),AU-12.1(ii),MA-4(1)(a)
```

File: shared/macros-ansible.jinja [hunk 2]
```diff
@@ -368,6 +369,7 @@ The macro requires following parameters:
     state: present
     dest: /etc/audit/audit.rules
     create: yes
+    mode: '0640'
```
Comment (vojtapolasek) [inline shared/macros-ansible.jinja:372]: Just a question, why this change? Is it related to ansible-lint?

**Review by vojtapolasek — CHANGES_REQUESTED** [review]
Hello, thank you very much for a new rule. Please see comments.

File: shared/macros-ansible.jinja (hunk 2 above)
Comment (teacup-on-rockingchair) [reply shared/macros-ansible.jinja:372]: Yep it is to make ansible-lint happy :)
